tests/test_receivers.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Receiver tests."""
+
+import json
+
+
+def test_send_wrong_pid(app, client, archiver, access_token_headers):
```

Comment:
not tests for the happy path?

It would be nice to add a test that will send data, and immediately after GET data.
I think that it should fail because it is missing the db.commit operation.

Reply:
The good ones were part of the resource tests, but I added one here too.
And it worked before because I used `db.session.add` but I changed it to the `ModelCommitOp`.